doc/source/conf.py

```diff
@@ -107,9 +107,10 @@
 # names to html links. The current version can link to the current location (i.e., do
 # nothing). For the other version, we just add a place-holder; its name and value are
 # unimportant because these versions will get replaced dynamically.
-html_theme_options = {}
-html_theme_options['versions'] = {version: ''}
-html_theme_options['versions']['[placeholder]'] = ''
+### Sam Rabin 2024-10-02: Commented out to resolve "WARNING: unsupported theme option 'versions' given". Might break things!
+### html_theme_options = {}
+### html_theme_options['versions'] = {version: ''}
+### html_theme_options['versions']['[placeholder]'] = ''
```

Comment:
@jasonb5 In trying to move us off that old `sphinx_rtd_theme` fork, I made changes based on your PR at ESMCI/cime#4613. However, I was getting "WARNING: unsupported theme option 'versions' given" warnings until I commented out this bit. I see that [CIME has it uncommented](https://github.com/ESMCI/cime/blob/98fc6648b68d8f619e3e17339572b2d4d685ec14/doc/source/conf.py#L119-L121), though. Does that warning still happen for CIME, or is there something better I can do to resolve it?